driver_test.go

```diff
@@ -1955,12 +1960,18 @@ func TestPodmanDriver_SignalTask(t *testing.T) {
 func TestPodmanDriver_Sysctl(t *testing.T) {
 	ci.Parallel(t)
 
+	// Test fails if not root
+	if os.Getuid() != 0 {
```

Comment:
Should this be [os.Geteuid](https://pkg.go.dev/os#Geteuid) or does it matter?

Reply:
I'm not sure if there's a case where the tests will change their UID so that EUID would be different from UID call. But it doesn't hurt to change it to Geteuid.